src/bp/ui-studio/src/web/components/Layout/index.tsx

```diff
@@ -22,7 +22,7 @@ import OneFlow from '~/views/OneFlow'
 import BotUmountedWarning from './BotUnmountedWarning'
 import GuidedTour from './GuidedTour'
 import LanguageServerHealth from './LangServerHealthWarning'
-import layout from './Layout.styl'
```

Comment:
Please correct me if I'm wrong as that's not my cup of tea but I think the goal was eventually to migrate to stylus because it's node native and sass isn't and it requires a dependency to a node binding. (is it good or bad .. I don't know, you tell me)

Reply:
I know nothing about stylus 😂 I've always worked with sass, I was just trying to get some consistency

Reply:
Personally I never really catched up with stylus and used to sass, so I don't really have an opinion here. I don't see a real incentive to make the switch